catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -28,9 +28,12 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <repository>mvn:ddf.features/utilities/${project.version}/xml/features</repository>
+
     <feature name="catalog-core-api" version="${project.version}"
              description="Catalog API interfaces and simple implementations.">
         <feature>security-core</feature>
+        <feature>action-core-impl</feature>
```

Comment:
Yes, this feature has more undeclared dependencies besides this one too. Before this change, if `action-core-impl` wasn't installed, this feature would fail to install.